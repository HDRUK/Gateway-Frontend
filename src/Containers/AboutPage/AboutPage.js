import React from "react";
import AppSideNav from "../../components/appSideNav/appSideNav";
import LandingContent from "../LandingContent/LandingContent";
import { SideNavPageWrapper } from "../../styles/styles";

const AboutPage = () => {
    return (
        <div>
            <AppSideNav />
            <SideNavPageWrapper>
                <LandingContent></LandingContent>
            </SideNavPageWrapper>
        </div>
    );
};

export default AboutPage;
