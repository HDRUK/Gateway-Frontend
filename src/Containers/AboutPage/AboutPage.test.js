import React from "react";
import { create } from "react-test-renderer";
import AboutPage from "./AboutPage.js";

describe("<AboutPage> ", () => {
    let renderedComponent;

    beforeEach(() => {
        renderedComponent = create(<AboutPage />);
    });

    it("should test the AboutPage Snapshot", () => {});
});
