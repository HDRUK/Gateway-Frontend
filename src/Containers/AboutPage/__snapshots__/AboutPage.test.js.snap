// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage>  should test the AboutPage Snapshot 1`] = `
<div>
  /about
</div>
`;
